resources/ibm-credentials.env

```diff
@@ -0,0 +1,4 @@
+VISUAL_RECOGNITION_APIKEY=1234abcd
```

Comment:
Do we really need this file in the core package???

Reply:
This is for unit tests.